front/src/components/boxs/weather/EditWeatherBox.jsx

```diff
@@ -22,6 +23,30 @@ const EditWeatherBox = ({ children, ...props }) => (
           ))}
       </select>
     </div>
+    <div className="form-group">
+      <div>
+        <label>
+          <Text id="dashboard.boxes.weather.editModeLabel" />
+        </label>
+      </div>
+      <div>
+        {Object.keys(GetWeatherModes).map(key => {
+          const mode = GetWeatherModes[key];
+          const label = 'dashboard.boxes.weather.displayModes.' + mode;
+          return (
+            <label>
+              <input
+                type="checkbox"
+                name={mode}
+                checked={props.box.modes !== undefined && props.box.modes[mode]}
+                onChange={props.updateBoxModes}
+              />
+              &nbsp; <Text id={label} />
```

Comment:
I'm not sure adding a `&nbsp;` char is expected. You may add margin with boostrap classes.

```
<span class="ml-2"><Text id={label} /></span>
```

https://getbootstrap.com/docs/4.0/utilities/spacing/

Reply:
+1

Reply:
Fixed